src/sentry/snuba/discover.py

```diff
@@ -22,6 +27,15 @@
 ReferenceEvent = namedtuple("ReferenceEvent", ["organization", "slug", "fields"])
 
 
+def is_aggregation(col):
+    match = AGGREGATE_PATTERN.search(col)
+
+    if match and match.group("function") in VALID_AGGREGATES:
```

Comment:
You should also return true if the col is in `FIELD_ALIASES` this list contains virtual columns we've added because we either can't support the required multi-parameter function or want to give higher level field names to users.

Reply:
Good point, updated to exclude these.